spec/controllers/user_spec.rb

```diff
@@ -24,6 +24,10 @@
 
         expect(response.status).to eq(302)
       end
+      it 'Logs the user in after sign up' do
+        post :create, params: user
+        expect(controller.logged_in?).to eq(true)
+      end
```

Comment:
I believe there are redirects you might want to test as well

Reply:
There is already a test for redirect at line 22 of this file